tests/devtools-evm-hardhat-test/hardhat.config.with-invalid-safe-networks.ts

```diff
@@ -0,0 +1,34 @@
+import 'hardhat-deploy'
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+import '@layerzerolabs/toolbox-hardhat'
+import { createTestNetworkConfigV2 } from '@layerzerolabs/test-setup-devtools-evm-hardhat'
+import type { HardhatUserConfig } from 'hardhat/types'
+
+const MNEMONIC = process.env.MNEMONIC ?? ''
+
+/**
+ * This is a dummy hardhat config that enables us to test
+ * hardhat functionality without mocking too much
+ */
+const config: HardhatUserConfig = {
```

Comment:
Here you can do what you originally did for the `validHre` / `invalidHre` and just import the base config and add what you need. That way you don't need to have any of the imports and/or other settings duplicated

```typescript
// ...
import baseConfig from './hardhat.config';

const config: HardhatUserConfig = {
  ...baseConfig,
  networks: {
    ...baseConfig.networks,
    // ...
  }
}
}
```